tests/behat/features/capabilities/security/private-file-uploads.feature

```diff
@@ -75,7 +75,7 @@ Feature: Private files
     And I attach the file "/files/opensocial.jpg" to "files[fid]"
     And I wait for AJAX to finish
     And I fill in "Alternative text" with "Just a private image test"
-    And I press "Save" in the "Modal footer"
+    And I press "Save" in the "WYSIWYG dialog"
```

Comment:
because @MaikelGG disabled the bootstrap modal. The WYSIWYG used to use it. Not anymore, so the behat needed some fixing.